tests/lib/SimpleSAML/Utils/Config/MetadataTest.php

```diff
@@ -259,4 +259,44 @@ public function testIsHiddenFromDiscovery()
             ],
         ]));
     }
+
+    
+    /**
+     * Test \SimpleSAML\Utils\Config\Metadata::parseNameIdPolicy().
+     */
+    public function testParseNameIdPolicy()
+    {
+        // Test null or unset
+        $nameIdPolicy = null;
+        $this->assertEquals(Metadata::parseNameIdPolicy($nameIdPolicy), ['Format' => \SAML2\Constants::NAMEID_TRANSIENT]);
```

Comment:
I think the `assert*()` methods receive the expected value first, then the actual value. It's really a minor detail, but could make it harder to debug when getting errors during testing, as the values would then be inverted.